src/Knp/Component/Pager/Paginator.php

```diff
@@ -198,7 +201,7 @@ public function connect(string $eventName, $listener, int $priority = 0): void
      */
     protected function dispatch(string $eventName, Event\Event $event): void
     {
-        if (!\class_exists(LegacyEventDispatcherProxy::class)) {
+        if (\class_exists(BaseEvent::class)) {
```

Comment:
Please revert this change. You can propose it in a separate PR.

Reply:
Hello. Okay, sounds better.